script/patch.py

```diff
@@ -62,6 +64,8 @@ def parse_args():
   parser = argparse.ArgumentParser(description='Apply patches to a git repo')
   parser.add_argument('-f', '--force', default=False, action='store_true',
                       help='Do not stop on the first failed patch.')
+  parser.add_argument('--project-root', required=False, default=git.get_repo_root(os.path.abspath(__file__)),
+                      help='Parent folder to resolve repos relative paths against')
   parser.add_argument('-R', '--reverse', default=False, action='store_true', help='Apply patches in reverse.')
   parser.add_argument('-r', '--repo', help='Path to a repository root folder.')
```

Comment:
Seems like `--repo` can serve the same purpose as `--project-root` in this script, can we remove one in favor of the other ?

Reply:
They are mutually exclusive, so technically yes.
But they have different semantics, so no =)

`--project-root` is used to explicitly set a parent folder for the `src/`, while `--repo` is used when a path to single patch is passed.